src/scratchtocatrobat/converter/test_converter.py

```diff
@@ -2153,12 +2153,26 @@ def test_can_convert_call_block_user_script_not_yet_defined_simple(self):
         assert expected_param_types == param_types
 
     #gotoSpriteOrMouse:
-    def test_can_convert_go_to_sprite_block_with_sprite(self):
-        test_sprite_name = "Abby"
+    def test_can_convert_go_to_sprite_block_with_sprite_afterwards(self):
+        test_sprite_name = "Afterwards"
         scratch_block = ["gotoSpriteOrMouse:", test_sprite_name]
+        context = converter.Context()
+        sprite_context = converter.SpriteContext(DUMMY_CATR_SPRITE.getName(), {})
+        sprite_context.context = context
+        script_context = converter.ScriptContext(sprite_context)
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE, script_context)
+        assert isinstance(catr_brick, catbricks.GoToBrick)
+        assert catr_brick.destinationSprite.getName() == test_sprite_name
+
+    #gotoSpriteOrMouse:
+    def test_can_convert_go_to_sprite_block_with_sprite_previous(self):
+        test_sprite_name = "Previous"
+        scratch_block = ["gotoSpriteOrMouse:", test_sprite_name]
+        sprite_object = SpriteFactory().newInstance(SpriteFactory.SPRITE_SINGLE, "Previous")
+        self.test_scene.spriteList.append(sprite_object)
         [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
         assert isinstance(catr_brick, catbricks.GoToBrick)
-        assert catr_brick.destinationSprite.name == test_sprite_name
+        assert catr_brick.destinationSprite.getName() == test_sprite_name
```

Comment:
This check is not sufficient here.
You may want to check object identity as well:
`assert catr_brick.destinationSprite is sprite_object`
Otherwise this test wouldn't differ from the previous one.